test/execution/execute-afm.test.ts

```diff
@@ -320,89 +782,225 @@ describe('executeAfm', () => {
             { status: 200, body: getPollingResponseBody() }
         );
 
-        const pagesByOffset: any = {
+        const pagesByOffset: IPagesByOffset = {
```

Comment:
could by extracted to separate fixture file

Reply:
I think that the test will be more readable if inputs and expects will be in the same file. Moreover, we would have to use helper functions for creating measures and attributes from this test file in fixtures file.